README.md

```diff
@@ -14,6 +14,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### 1.32.1
+*Released*: TBD
```

Comment:
I generally don't pin a version number until I start to do the actual release, but no big deal.